alibi/explainers/integrated_gradients.py

```diff
@@ -400,6 +400,14 @@ def _gradients_input(model: Union[tf.keras.models.Model],
 
     grads = tape.gradient(preds, x)
 
+    # if there are inputs have not impact to the output, the gradient is None, but we need to return a tensor
```

Comment:
Slight nit: Maybe "If certain inputs don't impact the target, the gradient is None, but we need to return a tensor"